src/v3/getRates.ts

```diff
@@ -19,7 +19,7 @@ const queryProviders = async (
   targetFiat: string,
   requestedCrypto: CryptoRateMap,
   requestedFiat: FiatRateMap,
-  rightNow: Date
```

Comment:
I would leave out this commit. This confuses the date that the request came in (rightNow) with the date the user requested. It made it much harder to review the next commit.